src/stan/lang/ast/node/cholesky_corr_var_decl_def.hpp

```diff
@@ -13,9 +13,9 @@ namespace stan {
 
     cholesky_corr_var_decl::cholesky_corr_var_decl(expression const& K,
                                        std::string const& name,
-                                       std::vector<expression> const& dims)
-      : base_var_decl(name, dims, MATRIX_T), K_(K) { }
-
+                                       std::vector<expression> const& dims,
+                                       expression const& def)
```

Comment:
same comment as above;  could you just find all the `const&` and rearrange?  Thanks.